.gitlab-ci.yml

```diff
@@ -113,9 +110,29 @@ test:
   needs:
     - check-std
   script:
-    - for crate in ${ALL_CRATES}; do
-        cargo test --verbose --all-features --release --manifest-path ${crate}/Cargo.toml;
-      done
+    - cargo test --verbose --all-features --no-fail-fast --release --workspace
+
+codecov:
+  stage:                           workspace
+  <<:                              *docker-env
+  needs:
+    - check-std
+  variables:
+    # according to https://github.com/mozilla/grcov/blob/master/README.md we have to use these variables, however
+    CARGO_INCREMENTAL:             0
+    # it does not work with -Zno-landing-pads
```

Comment:
do you know why it doesn't?

Reply:
Nope, it fails 6 tests with different errors, 2 of them are from `trybuild`